Cinnamenu@json: Update pt_BR.po

ViniciusQueirozVQ opened the pull request:
I corrected the translation file, using poedit, as directed by @fredcw  in  #3894 .

Miltonr87 commented:
> I corrected the translation file, using poedit, as directed by @fredcw in #3894 .

Hello friend, I noticed that your last character change disabled the translation of the expression "All Applications" to "Todos os aplicativos" when opening the Menu. The translation of this category was normal in the previous commit.

ViniciusQueirozVQ commented:
> > I corrected the translation file, using poedit, as directed by @fredcw in #3894 .
> 
> Hello friend, I noticed that your last character change disabled the translation of the expression "All Applications" when opening the Menu. The translation "Todos os aplicativos" was normal in the previous commit. I hope I helped with the feedback.

Oh, thanks for feedback! 
Fixed in next update. I'm opening a new pull request.
